backend/config/settings/local.py

```diff
@@ -3,7 +3,18 @@
 # Dummy value for development
 SECRET_KEY = "*m(r@4mdh*!zabwg&6tp%mgs_ezkprs9g+$@x@cdq-z_)dtf2i"
 
-DEBUG_TOOLBAR = False
+DEBUG_TOOLBAR = True
```

Comment:
I'd keep it False by default, having it on all the time has a significant performance impact.